string nome
float32 prezzo
string ingredienti
